TFT/src/User/API/HomeOffsetControl.c

```diff
@@ -48,8 +48,7 @@ float homeOffsetSetValue(float value)
 // Get current Z offset value
 float homeOffsetGetValue(void)
 {
-  z_offset_value = getParameter(P_HOME_OFFSET, Z_STEPPER);
-
+  z_offset_value = getParameter(P_HOME_OFFSET, AXIS_INDEX_Z);
```

Comment:
please maintain the empty line before "return". All my code use that syntax (as for the variable definition etc..)